servers/config/gateway/dynamic/fileConfig.yml

```diff
@@ -5,57 +5,55 @@ http:
         - http
       rule: 'Host(`localhost`)'
       middlewares:
-        - basic-auth
```

Comment:
please push this gateway configuration into your integration tests. Lets keep the basic authentication at the top-level for now. For a way to use a specific setup for tests, see `servers/lib/test/data` directory.